.github/workflows/create-dev-tag.yml

```diff
@@ -0,0 +1,86 @@
+name: Create Development Tag
+
+on:
+  workflow_call:
+    inputs:
+      branch:
+        description: 'The branch to create the tag on'
+        default: 'main'
+        required: true
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'The branch to create the tag on'
+        default: 'main'
+        required: true
+        type: choice
+        options:
+          - main
+          - release-2.1
+          - release-2.2
+          - release-2.3
+          - release-2.4
+          - release-2.5
+          - release-2.6
+
+  schedule:
+    - cron: '5 20 * * *'
+
+permissions:
+  contents: write
+
+env:
+  GOWORK: off
+  GOPRIVATE: github.com/mesosphere
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  GH_ACCESS_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
+
+jobs:
+  create-dev-tag:
```

Comment:
so we need to run this job on a cron for every release branch, is that what this job currently does? 🤔